### DISCUSSION TITLE
Which content type for a Canvas App

### DISCUSSION THREAD

**Discussion by FAC-1312** [microsoft/powerplatform-build-tools#741] [Q&A]
Hello everyone, 

I would like to add a Power Apps Canvas to a solution to export the solution. 
If I understood well I need to use this command to perform this action.

`pac solution add-solution-component --solutionUniqueName <MySolutionName> --component <Component> --componentType <Content-Type-of-CanvasApp>`

My question is: Which is the component type for a canvas app and wich is the component technical name for a canvas app? 

I Already try a Get-AdminPowerApp usinf Admin Power Platfrom cmdlets and get a look in internal properties but I didn't find nothing useful for this case. 

I appreciate any insights or assistance you can provide. Thank you in advance for your help!

**Comment by SiPurdy** [comment]
I have the same challenge with powerpagesites.
The complexity is that the componentType is environment specific so if the environment is fixed, you might be ok.

**Comment by craig-lund-bah** [comment]
https://learn.microsoft.com/en-us/power-apps/developer/data-platform/webapi/reference/solutioncomponent?view=dataverse-latest
